catanatron_core/catanatron/models/board.py

```diff
@@ -347,7 +361,12 @@ def is_enemy_road(self, edge, color):
 
 
 def longest_acyclic_path(board: Board, node_set: Set[int], color: Color):
-    global STATIC_GRAPH
+    if USING_FAST_GRAPH:
+        if type(color) != int:
+            color = color.value
```

Comment:
If you're using intEnum, why do you need color.value? (since each Enum instance is a subclass of int)

Reply:
Ah yea, good catch! I remember I did this and it was kinda hacky. I wanted to ensure that the conversion to the C++ binding wouldn't give any problems. I'll try it out without the .value call.

Reply:
Yep, looks like that works!